spec/factories/items.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :item do
+    name "MyString"
```

Comment:
Ah. Right now the name of the items created from the factory is going to be "MyString" -- you just need to update the "MyString" part to something a bit better, like an example of a real world task.